www/viewgame

```diff
@@ -1515,7 +1523,8 @@ function submitTime() {
         return;
     }
     // Time is valid, as far as we can tell
-    jsonSend("settime?game=<?php echo $id?>&newtime=" + new_time, "timeSaveMsg", swapTimeSections);
+    var time_note = document.getElementById('timeNote').value;
+    jsonSend("settime?game=<?php echo $id?>&newtime=${new_time}&note=${encodeURIComponent(time_note}), "timeSaveMsg", swapTimeSections);
```

Comment:
This line is a syntax error. I think you mean:

```
jsonSend(`settime?game=<?= $id?>&newtime=${new_time}&note=${encodeURIComponent(time_note)}`, "timeSaveMsg", swapTimeSections);
```

Reply:
Ok, I copy-pasted your code in there...hopefully I got it right this time.